teuthology/results.py

```diff
@@ -81,7 +82,10 @@ def results(archive_dir, name, email, timeout, dry_run):
                 body=body,
             )
     finally:
-        generate_coverage(archive_dir, name)
+        try:
+            generate_coverage(archive_dir, name)
+        finally:
+            Scraper(archive_dir).analyze()
```

Comment:
@jdurgin @grapheo12 I'm actually okay to merge it early if you revert these, line and submit integration part of scrape within another PR with test covering scrape.

Reply:
May I just comment this line and add ```pass``` statement?
Its all up to you. This is my first contribution to Ceph. :sweat_smile:
@kshtsk @jdurgin

Reply:
@grapheo12 Makes sense, also add a comment that this code is not tested and can be "enabled" after adding tests.